src/features/utility-bar/store.ts

```diff
@@ -45,24 +30,52 @@ export const screenSizeMap = {
   [ScreenSize.Desktop]: undefined,
 }
 
-export const useUtilityBarStore = create<UtilityState>()(
-  devtools((set) => ({
-    isSettingsVisible: false,
-    setIsSettingsVisible: (newVal: boolean) =>
-      set(() => ({ isSettingsVisible: newVal })),
+const getInitialMode = (): ThemeMode => {
+  const colorSchemeQuery = window.matchMedia('(prefers-color-scheme: dark)')
+  return colorSchemeQuery.matches ? 'dark' : 'light'
+}
```

Comment:
Doesn't need to be included now, but it would be nice to use `localStorage` here in addition to their theme preference. For instance, if I were to swap to `light` for parts-kit, I wouldn't want to have to do that each refresh.

Reply:
Zustand actually has a `persist` middleware that I'm using that does exactly that (line 39). 

https://docs.pmnd.rs/zustand/integrations/persisting-store-data